src/scr.c

```diff
@@ -878,7 +1024,14 @@ static int scr_get_params()
     scr_group = strdup(SCR_GROUP);
   }
 
+  if (scr_my_rank_world == 0) {
+     // 27. Dump SCR parameters - SCR_GROUP
+     scr_dbg(scr_debug, "SCR_GROUPO = %s", scr_group);
```

Comment:
Looks like there is an extra 0 character in the name here.
```suggestion
     scr_dbg(scr_debug, "SCR_GROUP = %s", scr_group);
```